examples/web-wagmi/src/publications/UseReaction.tsx

```diff
@@ -48,7 +50,7 @@ function ReactionButton({ publication, profileId, reactionType }: ReactionButton
 
   return (
     <>
-      <button onClick={toggleReaction} disabled={isPending}>
+      <button onClick={toggleReaction} disabled={isPending || (hasAnyReaction && !hasReactionType)}>
```

Comment:
only single type of reaction is allowed from the API side, so disable possibility of adding a reaction if another one is already present - pure UI solution